pubsub/pubsub_goredis.go

```diff
@@ -139,8 +153,18 @@ func (ps *GoRedisPubSub) Subscribe(ctx context.Context, topic string, callback f
 				if msg == nil {
 					continue
 				}
+				ctx, span := otelutil.StartSpanMulti(context.Background(), ps.Tracer, "GoRedisPubSub.Receive", map[string]interface{}{
```

Comment:
I'm slightly worried about the performance overhead of creating a context every time. How about creating just one outside the for loop and reusing it for each span?